lib/Test/Nginx/Socket.pm

```diff
@@ -2686,6 +2707,27 @@ will produce the following line (to C<stderr>) while running this test block:
 You need to remember to set the C<TEST_NGINX_NO_CLEAN> environment to 1 to prevent the nginx
 and other processes from quitting automatically upon test exits.
 
+=head2 response_body_filters
+
+Transform value for the body of the submitted request.
+
+    sub uppercase {
+        return uc $shift;
+    }
+    --- response_body_filters eval
+    \&main::uppercase
```

Comment:
hello @agentzh, i have trouble in use `\&uc`,  if i use `\&uc`, it will raise the error `Undefined subroutine &Test::Base::Filter::uc`, any idea to solve this?